src/v0/destinations/am/utils.js

```diff
@@ -110,6 +111,13 @@ const getUnsetObj = (message) => {
   return unsetObject;
 };
 
+const updateWithSkipAttribute = (message, payload) => {
+  const skipAttribute = get(message, 'integrations.Amplitude.skipUserPropertiesSync');
```

Comment:
Should we use the dedicated function for integrations object defined already here?


- just suggestion

Reply:
If we use that function, we must add amplitude to the canonical names. We will revisit refactoring amplitude soon. Then we can update this accordingly.